### PR TITLE
Feature/cypress test case heatmap

### PR DESCRIPTION
**Pull request by nidhisinghai**
### Description
cypress test case for heatmap Chart Styles

### Check List
- [ ] New functionality includes testing.
- [ ] All tests pass
- [ ]  Added cypress test case for Heatmap render including Newly added feature of 'Chart Style'

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

@abasatwar @spattnaik

### CODE REVIEW

**Review by abasatwar — CHANGES_REQUESTED** [review]

File: dashboards-observability/.cypress/integration/1_event_analytics.spec.js [hunk 1]
```diff
@@ -615,4 +615,87 @@ describe('Renders data view', () => {
...
+  });
+
+  it('Renders heatmap chart for Color Mode Spectrum and  Change color of Scheme', () => {
+    querySearch(TEST_QUERIES[4].query, TEST_QUERIES[4].dateRangeDOM);
+    cy.get('[data-test-subj="configPane__vizTypeSelector"] [data-test-subj="comboBoxInput"]').click();
+    cy.get('[data-test-subj="comboBoxOptionsList "] button span').contains('Heatmap').click();
+    cy.get('[aria-haspopup="true"]').eq(1).click();
+    cy.get('.euiColorPalettePicker__itemTitle').contains('Red-Blue').click();
+    cy.get('[data-test-subj="visualizeEditorRenderButton"]').click();
+    cy.get('stop[stop-color="rgb(5, 10, 172)"]').should('exist');
```
[the remark this answers is not in the document]
Comment (nidhisinghai) [reply dashboards-observability/.cypress/integration/1_event_analytics.spec.js:684]: The Assert is different for each of the Test case so can't be combined but whatever is common line of code is moved to a beforeEach function.

File: dashboards-observability/cypress.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "baseUrl": "http://localhost:5601",
+  "baseUrl": "http://localhost:5601/ryb",
```
[the remark this answers is not in the document]
Comment (nidhisinghai) [reply dashboards-observability/cypress.json:2]: No, i will remove this.

File: dashboards-observability/cypress.json [hunk 3]
```diff
@@ -16,7 +16,7 @@
   "experimentalNetworkStubbing": true,
   "env": {
     "opensearch": "localhost:9200",
-    "opensearchDashboards": "localhost:5601",
+    "opensearchDashboards": "localhost:5601/ryb",
```
[the remark this answers is not in the document]
Comment (nidhisinghai) [reply dashboards-observability/cypress.json:19]: No, i will remove this.

**Review by abasatwar — APPROVED** [review]

**Comment by nidhisinghai** [comment]
new separate files are created for charts under visualizationcharts folder

**Pull request closed without merging**